docs/api-client/README.md

```diff
@@ -1540,37 +1540,86 @@ GET /contract/prepareSignature
 
 **Parameters**
 
-| Name         | Type     | Description                           |
-| ------------ | -------- | ------------------------------------- |
-| `contractId` | `string` | **Required.** The ID of the contract. |
+| Name          | Type     | Description                                        |
+| ------------- | -------- | -------------------------------------------------- |
+| `contractId`  | `string` | **Required.** The ID of the contract.              |
+| `fullName`    | `string` | **Required.** The full name of the person signing. |
+| `accessToken` | `string` |  The access token representing the signatory. If none are specified, it will use the current user to determine who is signing right now. |
 
 Example
 
 ```
-GET /contract/prepareSignature?contractId=703e3f8c6e91fc7ba35633974ea96acab4c29c5ef17300bd6f5651ee53338487
+GET /contract/prepareSignature?contractId=703e3f8c6e91fc7ba35633974ea96acab4c29c5ef17300bd6f5651ee53338487&fullName=SomeName&accessToken=28394728947829374823723428742389462378423874
 ```
 
 ::: tip APIClient
 
 ```js
 apiClient.prepareSignature(
-  "703e3f8c6e91fc7ba35633974ea96acab4c29c5ef17300bd6f5651ee53338487"
+  "703e3f8c6e91fc7ba35633974ea96acab4c29c5ef17300bd6f5651ee53338487",
+  "My Full Name",
+  "34897234897289472894724(some access token)"
+);
+```
+
+:::
+
+**Response**
+
+Returns a promise which resolves with the signature. 
+
+Example
+
+```json
+{
+  "signature": "0x85fh946322957458934859038908903845ec5a7c4ad0f5c88e4076c65f3bb"
+}
+```
+
+### signContract
+
+Method used in to let openlaw take care of the entire signature process. This means that Openlaw will prepare the signature, create and send a transaction and keep track of its status.
```

Comment:
in to let -> to let Openlaw

Reply:
thank you guys! I've commented the other points. Let me know if there is anything else. I' not going to change anything because I don't want to create conflicts with the changes you have done @Jdville03  Let me know if I should do anything